internal/rbd/rbd_util.go

```diff
@@ -532,11 +534,18 @@ func (ri *rbdImage) isInUse() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	ri.Primary = mirrorInfo.Primary
+
+	if mirrorInfo.State == librbd.MirrorImageEnabled {
+		if !mirrorInfo.Primary {
```

Comment:
These could be combined into a single check for better readability and less indentation.